### DISCUSSION TITLE
1.x版本使用FormSlot，控制台有报warning

### DISCUSSION THREAD

**Discussion by henryzp** [alibaba/formily#1694] [Q&A 1.x] [answered]
![image](https://user-images.githubusercontent.com/3849702/123931692-556d2b00-d9c3-11eb-821d-eb157b00b8cd.png)

另外在api的导航页没有找到这个组件的用法。。我写的代码如下：

```jsx
<FormSlot>
  <h3 style={{ fontSize: 14 }}>派车详情</h3>
</FormSlot>
```

**Comment by janryWang** [comment, marked as answer]
因为FormSlot里用了x-render，本身也不推荐使用FormSlot的，最好用createVirtualBox

**Reply by henryzp** [reply]
有一个疑问是，为什么不提供一个简单的基于createVirtualBox的组件，就单纯渲染里面的children即可。。

**Reply by janryWang** [reply]
因为这种混写模式不是一个最佳实践
